content/sql/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'SQRT()'
+Description: 'Returns the square root of a given number.'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'SQL Server'
+CatalogContent: 
+  - 'learn-sql'
+  - 'path/analyze-data-with-sql'
+---
+
+**SQRT()** is a function in SQL that returns the square root of a number input. The square root of any given number is a number that multiplied by itself will result in the number. 
+
+## Syntax
+
+```sql
+SELECT SQRT(number);
```

Comment:
```suggestion
SQRT(number);
```
SELECT really isn't part of the syntax as it can be used in other clauses in the query as well.